commands/new.go

```diff
@@ -0,0 +1,169 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/go-morph/morph"
+	"github.com/spf13/cobra"
+
+	. "github.com/dave/jennifer/jen"
+)
+
+const baseDriverPath = "drivers"
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "new",
+		Short: "Bootstraps new resources",
+	}
+
+	cmd.AddCommand(
+		NewDriverCmd(),
+	)
+
+	return cmd
+}
+
+func NewDriverCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:           "driver",
+		Short:         "Generates necessary file structure for a new driver",
+		Args:          cobra.ExactArgs(1),
+		RunE:          newDriverCmdF,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+}
+
+func newDriverCmdF(cmd *cobra.Command, args []string) error {
+	driverName := args[0]
+	morph.InfoLogger.Printf("Generating necessary file structure for %q driver\n", driverName)
+
+	if _, err := os.Stat(baseDriverPath + "/" + driverName); !os.IsNotExist(err) {
+		return fmt.Errorf("driver %q already exists, skipping", driverName)
+	}
+
+	return generateNewDriver(driverName)
+}
+
+func generateNewDriver(driverName string) error {
+	morph.InfoLoggerLight.Printf("== CreateDriver: %q: generating driver files\n", driverName)
+	morph.InfoLoggerLight.Println("=================================================")
+
+	f := NewFile(driverName)
+
+	f.PackageComment("Initial code generated by generator.")
+
+	f.ImportNames(map[string]string{
+		"github.com/go-morph/morph/drivers": "drivers",
+		"github.com/go-morph/morph/models":  "models",
+	})
+
+	f.Var().Add(Id("driverName")).Op("=").Add(Lit(driverName))
+	f.Var().Add(Id("defaultConfig")).Op("=").Add(Op("&")).Id("Config").Values(Dict{
+		Id("MigrationsTable"): Lit("schema_migrations"),
+	})
+	f.Comment("add here any custom driver configuration")
+	f.Var().Add(Id("configParams")).Op("=").Add(Index().String().Values())
+
+	f.Func().Id("init").Params().Block(
+		Qual("github.com/go-morph/morph/drivers", "Register").Call(Lit(driverName), Id("&"+driverName).Values()),
+	)
+
+	f.Line()
+
```

Comment:
based on the other `f.Line()` calls, that are directly on top of the next block of code, I'd remove this whiteline